meta-mel-support/recipes-connectivity/connman/connman_1.28.bb

```diff
@@ -0,0 +1,13 @@
+require recipes-connectivity/connman/connman.inc
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
This is incorrect. FILESEXTRAPATHS is a mechanism provided to bbappends. The recipe already knows to look at the files dir next to the recipe. Just drop this line.